src/lib/components/private/QDocs.svelte

```diff
@@ -3,10 +3,23 @@
   import { QCard, QCardSection } from "$lib";
   import QApi from "./QApi.svelte";
   import type { QComponentDocs } from "$lib/utils";
-
-  export let componentDocs: QComponentDocs | QComponentDocs[];
-
-  $: isDark = $Quaff.dark.isActive;
+  import type { Snippet } from "svelte";
+
+  let {
+    children,
+    display,
+    pre,
+    usage,
+    componentDocs,
+  }: {
+    children?: Snippet;
+    display?: Snippet;
+    pre?: Snippet;
+    usage?: Snippet;
+    componentDocs: QComponentDocs | QComponentDocs[];
+  } = $props();
+
+  const isDark = $derived($Quaff.dark.isActive);
```

Comment:
Use the Quaff class (import from "$lib/classes/Quaff.svelte"): `$derived(Quaff.darkMode.isActive)`